executable/AndroidManifest.ksy

```diff
@@ -0,0 +1,187 @@
+meta:
+  id: android_manifest
+  title: Android Binary manifest xml
+  file-extension: xml
+  endian: le
+doc: |
+    Android manifest has its own binary xml format.
+    It is not documented.
+seq:
+  - id: hdr
+    type: header
+  - id: strings_table_info_size
+    size: 4
+    doc: |
+      Size of all fields related to strings table.
+      Size of strings table info often counts from 0x8 byte until
+      first byte of strings table.
+      It should be equal 7*4 + 4*strings_count. 
+      7 - is a number of previous fields (4 bytes size).
+      If it is not equal to that then there is an error in manifest strings offset 
```

Comment:
This implies that an integer size is stored here — why don't we use `type: u4` then?

Also, if it's a "size" of something in bytes — please note that per [KSY style guide](http://doc.kaitai.io/ksy_style_guide.html), it should be named `len_strings_table_info`.